### PR TITLE
Update office-365-user-view-access-to-project-and-roadmap.md

### PR DESCRIPTION
**Pull request by NadinMerali**

### CODE REVIEW

File: ProjectWeb/office-365-user-view-access-to-project-and-roadmap.md [hunk 1]
```diff
@@ -13,12 +13,13 @@ description: "Learn how certain Office 365 users can have view or limited edit a
...
-This provision will allow them to have limited access to projects and roadmaps that are shared with them, without needing to be assigned a Project Plan license.
+Applies to: Project for the web and Roadmap
 
-If your users need to be able to edit or create projects or roadmaps, you'll need to purchase the appropriate Project Plan license and assign it to the user.
+An appropriate Project subscription (Project Plan 1, Project Plan 3, or Project Plan 5) is required for your Microsoft 365 or Office 365 users to view a  [Roadmap](https://support.office.com/article/Video-Welcome-to-Roadmap-57764149-51b8-468f-a50d-9ea6a4fd835a) or a [Project for the web](https://go.microsoft.com/fwlink/?linkid=2108301) plan. Users assigned with certain Microsoft 365 or Office 365 subscriptions are allowed to have limited edit access to projects that are shared with them without needing to be assigned a Project subscription.
```
Comment (JohanFreelancer9) [inline ProjectWeb/office-365-user-view-access-to-project-and-roadmap.md:20]:
```suggestion
An appropriate Project subscription (Project Plan 1, Project Plan 3, or Project Plan 5) is required for your Microsoft 365 or Office 365 users to view a [Roadmap](https://support.office.com/article/Video-Welcome-to-Roadmap-57764149-51b8-468f-a50d-9ea6a4fd835a) or a [Project for the web](https://go.microsoft.com/fwlink/?linkid=2108301) plan. Users assigned with certain Microsoft 365 or Office 365 subscriptions are allowed to have limited edit access to projects that are shared with them without needing to be assigned a Project subscription.
```

File: ProjectWeb/office-365-user-view-access-to-project-and-roadmap.md [hunk 2]
```diff
@@ -51,7 +52,10 @@ Within each subscription, there are two apps that can be seen from the Microsoft
 - Project for Office (Plan X)
 - Dataverse
 
-The "Plan" in the title of Project for Office comes from the family of suite the app is included in. Both apps (from the same subscription) must be assigned to the user to view Project for the web or Roadmap.
+In the table below, the "Plan" in the title of "Project for Office" comes from the family of Microsoft 365/Office 365 suites that includes the “Project for Office” and “Dataverse” apps and directly applies to Project for the web and Roadmap. The “Project for Office (Plan X)” and “Dataverse” apps come with an appropriate Microsoft 365 or Office 365 subscription and provide the underpinnings to view and perform limited edits of plans built in Project for the web. These apps are not separate products and are only included within the Microsoft 365 or Office 365 subscriptions as outlined above in the Office 365 subscription with view access and Office 365 subscription with limited edit access sections.
+
+Additionally, both the Project for Office and Dataverse apps within the same subscription must be assigned to the user to 1. view or perform limited edits to plans built in Project for the web; or, 2. view a Roadmap.
```
Comment (JohanFreelancer9) [inline ProjectWeb/office-365-user-view-access-to-project-and-roadmap.md:57]:
```suggestion
In the table below, the "Plan" in the title of "Project for Office" comes from the family of Microsoft 365 and Office 365 suites that include the “Project for Office” and “Dataverse” apps and directly applies to Project for the web and Roadmap. The “Project for Office (Plan X)” and “Dataverse” apps come with an appropriate Microsoft 365 or Office 365 subscription and provide the underpinnings to view and perform limited edits of plans built in Project for the web. These apps are not separate products and are only included within the Microsoft 365 or Office 365 subscriptions as outlined previously in the Office 365 subscription with view access and Office 365 subscription with limited edit access sections.

Additionally, both the Project for Office and Dataverse apps within the same subscription must be assigned to the user to view or perform limited edits to plans built in Project for the web or to view a Roadmap.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NadinMerali, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by tiburd** [comment]
@NadinMerali
Hi,

This pull request was opened in the public repo. PMs and content developers should work in the private repo, as described in [the authoring guide](https://review.docs.microsoft.com/en-us/Office-Authoring-Guide/office-docs-authoring-overview?branch=main). 

We encourage you to use the private repo for future contributions to Microsoft Learn.

However, if you want to proceed with this pull request, please resolve the file conflict and then @ mention me.

Thanks!

**Comment by scanum** [comment]
@NadinMerali We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this PR.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**